special-interest-groups/sig-planner/roles-and-organization-management.md

```diff
@@ -0,0 +1,68 @@
+# Planner SIG Roles and Organization Management
+
+## Join and Promotion
+
+We mainly follow the instructions in [TiDB Developer Group](../../architecture/README.md#tidb-developer-group).
+It lists common requirements, responsibilities and privileges of each role in developer group.
+
+### Join
+
+You are welcomed to be a [contributor](../../architecture/README.md#contributor) once one of your pr about planner is merged.
+To have a discusstion with us, just join our slack channel #sig-planner in https://pingcap.com/tidbslack.
+
+### Promotion
+
+#### Be Promoted to [`Active Contributor`](../../architecture/README.md#active-contributor)
+
+SIG Specific Requirements:
+
+* Joined the sig-planner slack channel.
+* Have at least 5 PRs merged in the planner/statistics package within one year.
+
+#### Be promoted to [`Reviewer`](../../architecture/README.md#reviewer)
+
+SIG Specific Requirements:
+
+* Have at least 10 PRs merged in the planner/statistics package within one year.
+* Fix at least 2 issues whose difficulty is medium or above.
+
+#### Be promoted to [`Committer`](../../architecture/README.md#committer)
+
+SIG Specific Requirements:
+
+* Have at least 20 PRs merged in the planner/statistics package within one year.
+* Fix at least 10 issues whose difficulty is medium or above.
```

Comment:
Can we reduce the number of solved medium issues? It's too high, which makes a huge gap between `reviewer` and `committer` 😂